internals/attachments.py

```diff
@@ -105,11 +106,18 @@ def get_attachment(feature_id: int, attachment_id: int) -> Attachment|None:
   return None
 
 
-def get_attachment_uri(attachment: Attachment) -> str:
-  """Return the URI path that will serve tis attachment."""
-  uri = '/feature/%r/attachment/%r' % (
+def get_attachment_url(attachment: Attachment) -> str:
+  """Return the URL path that will serve this attachment."""
+  if settings.DEV_MODE or settings.UNIT_TEST_MODE:
+    origin = settings.SITE_URL
+  else:
+    digits = attachment.key.integer_id() % 1000
```

Comment:
Could you add a comment as to why we are doing modulo 1000?

Reply:
Done.